src/test/TestUtils.cpp

```diff
@@ -194,6 +197,78 @@ genesis(int minute, int second)
         getTestDateTime(1, 7, 2014, 0, minute, second));
 }
 
+ConfigUpgradeSetKey
+upgradeSorobanNetworkConfig(std::function<void(SorobanNetworkConfig&)> modifyFn,
+                            SimulationPtr simulation)
+{
+    auto nodes = simulation->getNodes();
+    auto& lg = nodes[0]->getLoadGenerator();
+    auto& app = *nodes[0];
+
+    auto& complete =
+        app.getMetrics().NewMeter({"loadgen", "run", "complete"}, "run");
+    auto completeCount = complete.count();
+    // Only create an account if there are none aleady created.
+    if (app.getMetrics()
+            .NewMeter({"loadgen", "account", "created"}, "account")
+            .count() == 0)
+    {
+        auto createAccountsLoadConfig =
+            GeneratedLoadConfig::createAccountsLoad(1, 1);
+        createAccountsLoadConfig.offset =
```

Comment:
this doesn't look right: offsets are used if multiple nodes are creating accounts at the same time, so we use offset as to not overlap created accounts. Here we're only using one node, so we shouldn't need to configure offsets (same applies to other loadgen modes below)

Reply:
The issue is that this function is used by tests that create accounts after calling `upgradeSorobanNetworkConfig`. If an offset is not used, the test case's subsequent account creation load generation will fail as the first `CREATE_ACCOUNT` operation will return `CREATE_ACCOUNT_ALREADY_EXIST`. The the load generation will not complete and the simulation will timeout.